echo/util/SAT.hx

```diff
@@ -495,7 +506,7 @@ class SAT {
       test1 = min1 - max2;
       test2 = min2 - max1;
 
-      // Preform another test
+      // Preform another test // TODO: What is this test doing, exactly?
```

Comment:
I couldn't articulate exactly what this test is doing, so a short descriptor might help future folks if they need to come in here for any reason.